src/components/m-virtual-displays.vue

```diff
@@ -276,7 +294,21 @@ export default {
     this.displayType = this.defaultDisplays[0];
   },
   methods: {
+    isDisabled(item) {
+      return this.disabledItems.some(disabledItem => disabledItem === item[this.keyField])
+    },
+    isSelected(item) {
+      return this.selectedRows.some(row => row[this.keyField] === item[this.keyField] )
+    },
+    onChangeSelectionRow(item) {
+      if (this.isSelected(item)) {
+        this.selectedRows = this.selectedRows.filter(row => row[this.keyField] !== item[this.keyField])
+      } else {
+        this.selectedRows.push(item)
+      }
 
+      this.$emit('on-change-selection-row', { selectedRows: this.selectedRows })
```

Comment:
Je passerai bien le `item` concerné pour faire une action dans l'app si besoin

```suggestion
      this.$emit('on-change-selection-row', { item, selectedRows: this.selectedRows })
```